src/main/java/org/broadinstitute/consent/http/util/JsonSchemaUtil.java

```diff
@@ -83,7 +85,9 @@ public boolean isValidSchema_v1(String datasetRegistrationInstance) {
           logger.trace("Validation error: " + m);
         }
       }
-      return false;
+      return ve.getAllMessages();
+    } catch (Exception e) {
```

Comment:
Are we able to catch anything more specific here?

Reply:
I considered sending back the error message, but I didn't want to send the user too many details if we're not sure what the exception is. I thought a generic error here was appropriate as such